testutils/inject/shell_service.go

```diff
@@ -0,0 +1,24 @@
+package inject
```

Comment:
The Shell service had no pre-existing injectable implementation in `inject`. I added one, but it has no injection function for `Shell`.